QRScanner/QRScannerView.swift

```diff
@@ -133,11 +133,6 @@ public class QRScannerView: UIView {
         session.inputs.forEach { session.removeInput($0) }
         session.outputs.forEach { session.removeOutput($0) }
         removePreviewLayer()
-        if session.isRunning {
-            videoDataQueue.async { [weak self] in
-                self?.session.stopRunning()
```

Comment:
This is good to avoid crash but I wonder session will stop.
Please let me check 🙏

Reply:
@daichiro FYI: I created a test branch to reappearance this issues. https://github.com/mercari/QRScanner/tree/issues_15_test

Reply:
@hitsubunnu Thanks! Your branch is really helpful!!
I confirmed `AVCaptureSession`'s `deinit` is called correctly.